lib/layers/presentation/search_user/search_user_page.dart

```diff
@@ -0,0 +1,97 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:wot_statistic/generated/l10n.dart';
+import 'package:wot_statistic/layers/presentation/search_user/bloc/search_user_cubit.dart';
+import 'package:wot_statistic/layers/presentation/statistic_page/statistic_page.dart';
+
+import 'package:wot_statistic/common/theme/text_styles.dart';
+import 'bloc/search_user_state.dart';
+import 'package:wot_statistic/injection_container.dart' as di;
+
+class SearchUserPage extends StatelessWidget {
+  const SearchUserPage({Key? key, required this.heroTag}) : super(key: key);
+  final String heroTag;
+
+  @override
+  Widget build(BuildContext context) {
+    Size size = MediaQuery.of(context).size;
+    return Hero(
+      tag: heroTag,
+      child: Center(
+        child: SizedBox(
+          height: 3 * size.height / 4,
```

Comment:
could you explain, what is the strange mathematic there? By the way, if 3 and 4 will be moved to variables, undertand it will be much easier